packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const get = require('lodash/get');
-const { parseSQSMessageBody } = require('@cumulus/aws-client/SQS');
+const { parseSQSMessageBody, sendSQSMessage } = require('@cumulus/aws-client/SQS');
```

Comment:
The changes here are intended to explicitly push errors to the queue/lambda DLQ - the assumption that 'meta.collection' would have a value provided in the message template is buried pretty deeply in a lot of workflows and so it's entirely possible with this change to the Rule schema to misconfigure a rule that results in granule ingest that *fails* to write granules, but the workflow succeeds, the record is exported to CMR, etc.